### PR TITLE
JumpEffect and rendering improvements

### PR DESCRIPTION
**Pull request by Ell1ott**
- Added drawGradientQuad and drawGradientCircle (see usage example in ModuleJumpEffect.kt) functions to RenderShortcuts
- Made curve setting work like a drop-down where you can choose the preferred curve type, and added some different curve types: Linear, EaseIn, EaseOut, EaseInOut
- Added jumpEffect module. It adds a subtle but nice circle effect when you jump. I will send a video showcase of it on the guildet server.

image

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleJumpEffect.kt [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+object ModuleJumpEffect : Module("JumpEffect", Category.RENDER) {
+    private val endRadius by floatRange("EndRadius", 0.15F..0.8F, 0F..3F)
+
+    private val innerColor by color("InnerColor", Color4b(0, 255, 4, 0))
+    private val outerColor by color("OuterColor", Color4b(0, 255, 4, 89))
+
+    private val animCurve by curve("AnimCurve", Curves.EASE_OUT)
+
+    private val hueOffsetAnim by int("hueOffsetAnim", 63, -360..360)
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleJumpEffect.kt]: To make it fit with the capitalisation of the rest, please name it "HueOffsetAnim".

Comment (Ell1ott) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleJumpEffect.kt]: Yes, I missed that. Thanks for catching it

**Comment by RtxOP** [comment]
this guy is making pull reqs like crazy

**Comment by ghost** [comment]
Certified professional at work 🔥 💪

**Comment by EclipsesDev** [comment]
keep up the good work 🫡

**Comment by Ell1ott** [comment]
Will do! 🫡

**Pull request merged**